### PR TITLE
RHELAI-429: Adding upgrade informer service

### PR DESCRIPTION
**Pull request by javipolo**
This is a duplication of https://github.com/containers/ai-lab-recipes/pull/663, since @tsorya is on PTO and we really need to push those changes

/cc @rhatdan 
/cc @jeremyeder

Upgrade informer will run every couple of our and will be triggered by systemd timer.

In order to start it on boot and run once i enabled it and timer. Disabling auto upgrade service in order to remove unexpected reboots. Service will run "bootc upgrade --check" and in case new version exists it will create motd file with upgrade info.

### CODE REVIEW

File: training/common/usr/lib/systemd/system/upgrade-informer.service [hunk 1]
```diff
@@ -0,0 +1,12 @@
+[Unit]
+Description=Check for available operating system updates
```
Comment (romfreiman) [inline training/common/usr/lib/systemd/system/upgrade-informer.service]: s/operating system/RHEL AI/ upgrade ?

**Comment by rhatdan** [comment]
This should run only once per day.  If you run it more often then that you will see complaints about `flooding the logs`.

We are not going to be updating that often that it requires more then once/day to run.

**Comment by javipolo** [comment]
> This should run only once per day

Agree, changed the timer to `1day`

**Pull request merged**